consensus/ibft/consensus_backend.go

```diff
@@ -76,52 +71,68 @@ func (i *backendIBFT) InsertBlock(
 		return
 	}
 
-	if err := i.runHook(InsertBlockHook, header.Number, header.Number); err != nil {
-		i.logger.Error("cannot run hook", "name", string(InsertBlockHook), "err", err)
-
-		return
-	}
-
 	i.updateMetrics(newBlock)
 
 	i.logger.Info(
 		"block committed",
 		"number", newBlock.Number(),
 		"hash", newBlock.Hash(),
-		"validators", len(i.activeValidatorSet),
+		"validators", i.currentValidators.Len(),
 		"committed", len(committedSeals),
 	)
 
+	if err := i.currentHooks.PostInsertBlock(newBlock); err != nil {
+		return
```

Comment:
Maybe add an error log here just for consistency

Reply:
Added 3c33cb0cb1f902ec81cd3d708a8a239fa02aef2e